src/components/movie-view/movie-view.jsx

```diff
@@ -1,4 +1,4 @@
-import PropTypes from "prop-types";
+
```

Comment:
I noticed that your previous components had propTypes but in this submission you are deleting them. Any reason why?